wordpress-mobile/WordPress-Android
# Fix #5340: Fix media in legacy editor drafts and preview (FluxC)

Fixes #5340, restoring image display in the legacy editor for existing drafts and in legacy preview.

Note: Once we drop the legacy editor, we can drop the `MediaStore`-dependent code in `HtmlToSpannedConverter` (as well as its FluxC connectedness). `WPHtml` will still be used by `CommentAdapter`, but comments never contain local media.

To test:
1. Start a new draft in the legacy editor
2. Insert a local image
3. Exit the editor, saving the post as a draft
4. Re-open the draft
5. The image should be displayed
6. Preview the post (using the preview button only available in the legacy editor) - the image should be displayed

cc @maxme
